decidim-core/lib/decidim/common_passwords.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Decidim
+  class CommonPasswords
+    include Singleton
+
+    attr_reader :passwords
+
+    URLS = %w(
+      https://raw.githubusercontent.com/danielmiessler/SecLists/master/Passwords/xato-net-10-million-passwords-1000000.txt
+      https://raw.githubusercontent.com/danielmiessler/SecLists/master/Passwords/darkweb2017-top10000.txt
+      https://raw.githubusercontent.com/danielmiessler/SecLists/master/Passwords/Common-Credentials/10-million-password-list-top-1000000.txt
+    ).freeze
+
+    def initialize
+      update_passwords! unless File.exist?(password_list_path)
```

Comment:
Can this happen in real cases that the file doesn't exists as it is included with the Decidim release and codebase?

Maybe we should:
1. Make the `initialize` method raise an error (`FileNotFoundError`) when the file does not exist
2. Make the `update_passwords!` a class method so it wouldn't have to call the `initialize` method